### ISSUE TITLE
Slow query when loading the community pages

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#8689]
### Expected behavior
The community pages should load in a reasonable time.

### Actual behavior
It takes up to one minute until the page is loaded.

Database: 32.649/0.135, Network: 0, Rendering: 0.01, Session: 0, I/O: 0, Other: 0.2, Total: 33

**From my slow queries log:**
```
# Time: 200526 16:40:17
# User@Host: ...
# Thread_id: 1328468  Schema: friendica  QC_hit: No
# Query_time: 32.238983  Lock_time: 0.000018  Rows_sent: 20  Rows_examined: 22327397
# Rows_affected: 0  Bytes_sent: 562
# Tmp_tables: 1  Tmp_disk_tables: 0  Tmp_table_sizes: 3435424
# Full_scan: Yes  Full_join: No  Tmp_table: Yes  Tmp_table_on_disk: No
# Filesort: Yes  Filesort_on_disk: No  Merge_passes: 0  Priority_queue: Yes
SET timestamp=1590511217;
SELECT `name` AS `term`, COUNT(*) AS `score` FROM `tag-search-view` WHERE `private` = 0 AND `received` > DATE_SUB(NOW(), INTERVAL 24 HOUR) GROUP BY `term` ORDER BY `score` DESC LIMIT 20;
```

**What 'explain' tells me:**
```
MariaDB [friendica]> explain SELECT `name` AS `term`, COUNT(*) AS `score` FROM `tag-search-view` WHERE `private` = 0 AND `received` > DATE_SUB(NOW(), INTERVAL 24 HOUR) GROUP BY `term`
 ORDER BY `score` DESC LIMIT 20;
+------+-------------+----------+-------+---------------+---------------+---------+---------------------------+--------+----------------------------------------------+
| id   | select_type | table    | type  | possible_keys | key           | key_len | ref                       | rows   | Extra                                        |
+------+-------------+----------+-------+---------------+---------------+---------+---------------------------+--------+----------------------------------------------+
|    1 | SIMPLE      | tag      | index | PRIMARY       | type_name_url | 643     | NULL                      | 600670 | Using index; Using temporary; Using filesort |
|    1 | SIMPLE      | post-tag | ref   | PRIMARY,tid   | tid           | 4       | friendica.tag.id          |      7 | Using where; Using index                     |
|    1 | SIMPLE      | item     | ref   | uri-id        | uri-id        | 5       | friendica.post-tag.uri-id |      1 | Using where                                  |
+------+-------------+----------+-------+---------------+---------------+---------+---------------------------+--------+----------------------------------------------+
```

When I load the page again after a couple of minutes (by clicking on the community page icon) the page is loaded in less than one second. 

### Steps to reproduce the problem
Click on the community page icon to go to the global community page. The issue also exists for the local community page.

### Friendica version you encountered the problem
2020.06-dev

### Friendica source (git, zip)
git

### PHP version
7.3

### SQL version
MariaDB 10.3

**Comment by annando** [comment]
This looks like it is generated by the tag cloud or something similar. This is data that we really shouldn't create "on the fly", but every few minutes in the background.

**Comment by AlfredSK** [comment]
I can try to disable that feature and retest it.

**Comment by AlfredSK** [comment]
Ah! I guess the trending tags are created when I load the page for the first time. Later they're simply reused or loaded from buffer/cache. That would make sense.

**Comment by ben-utzer** [comment]
> Later they're simply reused or loaded from buffer/cache. That would make sense.
This could be loaded in the background once an hour or so. 

I also see this delay on the first loading of the page.

**Comment by annando** [comment]
The trending tags need to be coded differently. Currently I can only suggest to deactivate them.

**Comment by AlfredSK** [comment]
Duplicate of #9276 
Fixed by #9278 

Closing.
